src/Dapr.AzureFunctions.Extension/DaprExtensionConfigProvider.cs

```diff
@@ -117,7 +117,12 @@ static DaprPubSubEvent CreatePubSubEvent(object arg)
 
         static DaprPubSubEvent CreatePubSubEvent(JObject json)
         {
-            DaprPubSubEvent e = json.ToObject<DaprPubSubEvent>();
+            DaprPubSubEvent? e = json.ToObject<DaprPubSubEvent>();
+            if (e == null)
+            {
+                throw new Exception("Unable to deserialize pub/sub event.");
```

Comment:
It's not considered good practice to throw `Exception` directly. Consider using `ArgumentException` instead since the problem is specific to the `json` argument that was provided by the caller.